### PR TITLE
add scroll alpha testnet

### LINKED ISSUE #2310: Add support for `Scroll Alpha Testnet`.
chain-id - 534353
rpc:https://alpha-rpc.scroll.io/l2
api: https://blockscout.scroll.io/api
explorer: https://blockscout.scroll.io/

### PR DESCRIPTION
**Pull request by salman01zp**
## PR Checklist

-   [ ] Added Tests
-   [ ] Added Documentation
-   [ ] Breaking changes

---
- closes #2310

### CODE REVIEW

File: ethers-core/src/types/chain.rs [hunk 1]
```diff
@@ -412,6 +414,10 @@ impl Chain {
                 ("https://blockscout.com/xdai/mainnet/api", "https://blockscout.com/xdai/mainnet")
             }
 
+            ScrollAlphaTestnet => {
+                ("https://blockscout.scroll.io/api", "https://blockscout.scroll.io/")
+            }
```
Comment (mattsse) [inline ethers-core/src/types/chain.rs:419]: is this right?

kinda weird, because this looks more like the mainnet URL?

Comment (salman01zp) [reply ethers-core/src/types/chain.rs:419]: yeah.. but it is for scroll-alpha-testnet [https://blockscout.scroll.io/](https://blockscout.scroll.io/)

You can also confirm with the scroll official page [here](https://scroll.io/alpha)

Comment (mattsse) [reply ethers-core/src/types/chain.rs:419]: very cursed imo

will cause issues once they replace it with mainnet or they add a `mainnet` to the path or domain, which would be even more confusing -.-

but looks like this is correct (at least for now), so lgtm

**Review by mattsse — APPROVED** [review]

**Comment by prestwich** [comment]
Please rebase onto latest main and then we can merge 👍

**Pull request merged**